web/pages/components/Navbar.tsx

```diff
@@ -0,0 +1,199 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Menu,
+  MenuButton,
+  MenuList,
+  Image,
+  Stack,
+  Text,
+  Link,
+  AccordionButton,
+  AccordionItem,
+  Accordion,
+  AccordionPanel,
+  Divider,
+} from "@chakra-ui/react";
+import { ChevronDownIcon, HamburgerIcon } from "@chakra-ui/icons";
+import { useAuth } from "../../context/auth";
+import { trpc } from "../../utils/trpc";
+export default function Navbar() {
+  const { user } = useAuth();
+  const userRole = trpc.user.getRole.useQuery({ uid: user?.uid ?? null });
+  if (user !== null) {
+    return (
+      <Flex
+        bgColor="white"
+        width="100%"
+        maxH="70px"
+        position="absolute"
+        top="0%"
+        bottom="90%"
+        height="auto"
+      >
+        <Stack
+          display={["none", "flex"]}
+          direction="row"
+          width="50%"
+          alignItems="center"
+          paddingLeft={2}
+        >
+          <Image
+            src="https://angelsrescue.org/wp-content/uploads/2020/05/A-Mark.svg"
+            alt="logo"
+            boxSize={[8, 10]}
+          ></Image>
+        </Stack>
+        <Stack
+          display={["none", "flex"]}
+          width="50%"
+          justifyContent="flex-end"
+          direction="row"
+          alignItems="center"
+          spacing={10}
+        >
+          {userRole.data == "admin" && (
```

Comment:
Use strict equality (`===`) and `Role.Admin` type instead of checking against a string literal.